Articles/Blog/2023-03-08-productivity-practices-and-tools.md

```diff
@@ -0,0 +1,155 @@
+
+At longer cadences, more substantial reflection and review is appropriate.
+For example, at monthly intervals, its reasonable to check in with yourself and ask...what should I stop, start, and continue for next month?
+Are my practices productive and effective or do I need to make some adjustments?
+At quarterly intervals, ask yourself...what is my vision/theme/aspiration(s) for this season?
+How well did I live up to my aspirations last quarter?
+
+#### Tools to consider
+
+There is a video series from [Tiago Forte](https://www.youtube.com/playlist?list=PLVNXAaej57W7fbYoc_XJ0bA0T3k_Ye-Q2) that describes a variety of journaling tools.
+In addition, consider [Drafts](https://getdrafts.com/), [GoodNotes](https://www.goodnotes.com/), [Notion](https://www.notion.so/), [Obsidian](https://obsidian.md/), [Evernote](https://evernote.com/).
+
+### Task Management
+
+While journaling aims to help *you* to be a generally more productive person, task management enables productivity of a particular plan of action.
+If tasks are not managed, they can wind up being left undone or, maybe worse, done without respect for priorities.
+In turn, this can hurt longer term goals.
+
+For many, the tactile act of checking tasks off as done or moving tasks across a project board (e.g. Kanban) has great appeal because it provides a sense of progress.
+
+#### What goes in to task management?
+
+When defining tasks, make them SMART -- specific, measurable, attainable, relevant, and timely.
+
+* Specific -- Specify as many of the [Five W's](https://en.wikipedia.org/wiki/Five_Ws) as possible (Who is responsible? What is the definition of done? When does it need to be completed? Where does it need to happen? Why is it necessary?)
+* Measurable -- Identify a quantifieable way for an observer to judge progress or completion. What are the artifacts or evidence of progress or completion?
+* Achievable -- Is everything needed (experience, skills, time, resources, etc.) to accomplish the task available? This may not always be clearly known before hand.
+* Relevant -- Is the task related to broader goals and themes?
+* Time-Bound -- What is the deadline? Or, what is the the maximum time you are willing to put into this task to move it forward?
+
+This information can help determine know how a task fits into larger goals as well as when and how it should be done.
+
+Another useful tool for managing tasks is the [*Eisenhower Matrix*](https://en.wikipedia.org/wiki/First_Things_First_(book)) which divides tasks along two axes: "urgent and not urgent" and "important and not important".
+The approach suggests that you **DO** those tasks that are both urgent and important, **PLAN** those tasks that are not urgent but still important, **DELEGATE** those tasks that are urgent but not important and **ELIMINATE** those tasks that are neither urgent nor important.
+
+#### Tools to consider
+
+If your work life is many short tasks that come and go quickly, consider a white board.
```

Comment:
I vehemently disagree with this.  I use a whiteboard to manage very long running tasks.  I've also used pencil and paper for this.  I've even used a shared whiteboard in a software development team.  Yes, I will admit that a whiteboard doesn't scale to a point.  If you have a project the scope of ECP, you need something like Jira.  There are too many team members, that are too distributed, working on tens of thousands of tasks.  However, a whiteboard can scale to 30-80 tasks just fine.

Another piece of nuance that has been lost here is that you can use multiple tools for different things.  For example the whiteboard works well for projects, but doesn't do as well at reminding me that I need salsa from the grocery store -- I use Reminders for that.

Reply:
Not sure there is really enough here to be in *vehement* disagreement :wink:. 

Given the article's current length, I don't think all the nuance here needs to be captured.

Fixed in 7513fa8b69652c66c0b3b32120be2d9c01538e5f

Reply:
My objection was that the prior phrasing leaves the impression that whiteboards (and by implication pencil and paper, because almost anything you can do on a whiteboard you can do with paper), are "second class citizens" to digital tools relegated people with only short tasks to do.  As knowledge workers most of the tasks that we do are long.  Therefore I read this as whiteboards aren't useful for knowledge workers.  That's where my objection was coming from. *vehement* is probably a strong phrasing, but I really do think that 80+% of people or more should start with and as many as 50+% should stick with analogue tools.

I'm open to leaving the nuance about multiple tools out.  I just wanted to note it here.